projects/plugins/jetpack/modules/carousel/jetpack-carousel.js

```diff
@@ -727,7 +736,7 @@
 				captionHtml = domUtil.convertToPlainText( current.attrs.caption );
 
 				if ( domUtil.convertToPlainText( current.attrs.title ) === captionHtml ) {
-					var title = carousel.info.querySelector( '.jp-carousel-photo-title' );
+					var title = carousel.info.querySelector( '.jp-carousel-photo-caption' );
 					domUtil.fadeOut( title, function () {
```

Comment:
If I'm understanding correctly, this is trying to hide the caption underneath the photo if its text is identical to the title? It looks like previously we hid the title in the extended info section instead of the caption. Nit -- if we keep this, it would be clearer to name this variable `caption`.

I tested this by setting the title, caption, and description to identical text. The description was correctly hidden, but the title and caption both displayed in the extended info section, and the caption under the photo displayed as well 🤔 I wonder if it would make more sense to hide the `.jp-carousel-caption` (the one in the extended info section) when the title and caption match?

Reply:
I think I've fixed this now. Good catch. I've completely removed this bit of code since it was conflicting with the function that shows/hides the other data. I think it's ready for another look.